src/MongoDB/ClientEncryption.stub.php

```diff
@@ -41,7 +41,7 @@ final class ClientEncryption
 
     final public function __construct(array $options) {}
 
-    final public function createDataKey(string $kmsProvider, array $options = []): \MongoDB\BSON\Binary {}
+    final public function createDataKey(string $kmsProvider, ?array $options = null): \MongoDB\BSON\Binary {}
```

Comment:
Was it necessary to change the default value from `[]` to `null`? Or was this mainly to be consistent with the use of `Z_PARAM_ARRAY_OR_NULL` in the method implementation?

I had used the following for in my proposed changes for https://github.com/mongodb/mongo-php-driver/pull/1339:
```
#if PHP_VERSION_ID >= 80000
    final public function rewrapManyDataKey(array|object $filter, ?array $options = []): object {}
#else
    /** @param array|object $filter */
    final public function rewrapManyDataKey($filter, ?array $options = []): object {}
#endif
```

This satisfied a PHPT test I created with an explicit `null` passed as `$options`. I'm amenable to changing the stub file to use `null` as you have here to be consistent, but just wanted to confirm whether it was significant.

Reply:
This was mainly done for consistency, as omitting the parameter and passing an explicit `null` has the same effect.